core-developers/experts.rst

```diff
@@ -62,7 +62,7 @@ argparse
 array
 ast                   benjaminp, pablogsal, isidentical
 asynchat              josiahcarlson, giampaolo*, stutzbach^
-asyncio               1st1, asvetlov
+asyncio               1st1, asvetlov, gvanrossum, graingert, kumaraditya303
```

Comment:
```suggestion
asyncio               1st1, asvetlov, gvanrossum*, graingert, kumaraditya303
```
You can an `*` after your name to indicate that you are fine with the issue being assigned to you.

Reply:
But I am not. I read everything and then decide.